src/Wallabag/CoreBundle/Entity/Config.php

```diff
@@ -17,7 +17,6 @@
  *         @ORM\Index(name="config_feed_token", columns={"feed_token"}, options={"lengths"={255}}),
  *     }
  * )
- * @ORM\Entity
```

Comment:
Why ?

Reply:
Because it's defined few lines above https://github.com/wallabag/wallabag/pull/4084/files/1576905e0357ef96cd852ea3c1ff4216af195c34#diff-8510db91e957b1978c2d3da4683ee547R13

Reply:
Indeed